_redirects

```diff
@@ -102,3 +102,5 @@
 # Bot Workshop
 /bot-workshop /assets/discord_bot_workshop.pdf
 
+# FOSDEM subsidy
+/fosdem https://docs.google.com/forms/d/e/1FAIpQLSdjcJy71Pbmywc-akGCbBdLrbq-kjbyXloNXbP3b9386A-0vg/viewform
```

Comment:
"The form FOSDEM Subsidy Request Form is no longer accepting responses."

Is this intentional for now?

Reply:
looks fine to me